cli/client_test.go

```diff
@@ -1001,3 +1001,31 @@ func TestShellFileCopy(t *testing.T) {
 		})
 	})
 }
+
+func TestUpdateOAuthAppAction(t *testing.T) {
```

Comment:
Please add cases which check that passing in bad values to each of these will result in a specific error message